node-json-db/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for node-json-db
+// Project: https://github.com/Belphemur/node-json-db
+// Definitions by: Ilya Kuznetsov <https://github.com/kuzn-ilya>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'node-json-db' {
```

Comment:
Could you please clarify exactly, you want me to get rid of using 'declare module'? So, I should just remove it and the result will be as below.

<code>
declare class JsonDB {

...

}

export = JsonDB;
</code>